jooby-servlet/src/main/java/org/jooby/servlet/ServletServletRequest.java

```diff
@@ -120,6 +116,15 @@ public String rawPath() {
   }
 
   @Override
+  public Map<String, Object> attributes() {
+    final Enumeration<String> attributeNames = req.getAttributeNames();
+    if (!attributeNames.hasMoreElements()) {
+      return Collections.emptyMap();
+    }
+    return Collections.list(attributeNames).stream().collect(Collectors.toMap(name -> name, name -> req.getAttribute(name)));
```

Comment:
split this line and leave one method call per line. Also for `name -> name`, use `Function.identity()`